pages/donate/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, Center, Container, Flex, Link, Text } from '@chakra-ui/react';
+
+const Donate: React.FC = () => {
+  return (
+    <>
+      <Flex gap={'150px'} flexDir={'column'}>
+        <Container maxW={'container.lg'} mb="100px">
+          <Flex gap={'60px'} flexDir="column">
+            <div data-cy="donate-top-message" id={'donate-top-message'}>
+              <Flex flexDir="column" gap={'15px'}>
+                <Center>
+                  <Box fontSize="5xl">Any amount helps us reach our goal!</Box>
+                </Center>
+              </Flex>
+            </div>
+            <div data-cy="donate-top-submessage" id={'donate-top-submessage'}>
+              <Flex flexDir="column" gap={'15px'}>
+                <Center>
+                  <Text fontSize="24px">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+                    incididunt ut labore et dolore magna aliqua
+                  </Text>
+                </Center>
+              </Flex>
+            </div>
+            <div data-cy="donate-button">
+              <Center>
+                <Link href="/">
+                  <Box
```

Comment:
What is the Box for? Can these styles not go directly into the Link?

Reply:
yes that works, done